aws/sdk/integration-tests/s3/Cargo.toml

```diff
@@ -21,7 +21,7 @@ aws-credential-types = { path = "../../build/aws-sdk/sdk/aws-credential-types",
 aws-http = { path = "../../build/aws-sdk/sdk/aws-http" }
 aws-runtime = { path = "../../build/aws-sdk/sdk/aws-runtime", features = ["test-util"] }
 aws-sdk-s3 = { path = "../../build/aws-sdk/sdk/s3", features = ["test-util", "behavior-version-latest"] }
-# aws-sdk-sts = { path = "../../build/aws-sdk/sdk/sts" }
+aws-sdk-sts = { path = "../../build/aws-sdk/sdk/sts" }
```

Comment:
Looks like this isn't used by the new stalled-stream-protection test?

Reply:
oops, removed